test/runtests.jl

```diff
@@ -25,11 +25,7 @@ include("test_utils/AllUtils.jl")
         Turing.setadbackend(adbackend)
         @testset "inference: $adbackend" begin
             @testset "samplers" begin
-                # FIXME: DynamicHMC version 1 has (??) a bug on 32bit platforms (but we were too
-                # lazy to open an issue so Tamas doesn't know about it), retest with 2.0
-                if Int === Int64 && Pkg.installed()["DynamicHMC"].major == 2
-                    include("contrib/inference/dynamichmc.jl")
-                end
+                include("contrib/inference/dynamichmc.jl")
```

Comment:
`Pkg.installed` is deprecated on Julia 1.4. I assume that it is unlikely that anyone will run the tests with DynamicHMC < 2 and according to the comment the bug (?) was observed on 32bit with DynamicHMC < 2, so I guess it should be safe to remove the check completely.

Reply:
Happy to remove this check.